sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/TrackNotImplementedException.cs

```diff
@@ -32,6 +32,14 @@ namespace SonarAnalyzer.Rules.CSharp
     [Rule(DiagnosticId)]
     public sealed class TrackNotImplementedException : SonarDiagnosticAnalyzer
     {
+        /// <remarks>
+        /// The ThrowExpressionSyntax and corresponding type, are not avialable
+        /// in the used version of Microsoft.CodeAnalysis.Charp 1.3.2.
+        ///
+        /// See: https://docs.microsoft.com/en-us/dotnet/api/microsoft.codeanalysis.csharp.syntax.throwexpressionsyntax?view=roslyn-dotnet
+        /// </remarks>
+        private const SyntaxKind ThrowExpression = (SyntaxKind)9052;
```

Comment:
Please use [SyntaxKindEx.ThrowExpression](https://github.com/SonarSource/sonar-dotnet/blob/8.5.0.15942/sonaranalyzer-dotnet/src/SonarAnalyzer.CFG/ShimLayer/SyntaxKindEx.cs#L47) - we use the `ShimLayer` which is borrowed from StyleCopAnalyzers to support C#7 + syntaxes

Reply:
I was not aware, and like the suggestion.